packages/renderer/src/lib/image/BuildImageFromContainerfile.svelte

```diff
@@ -155,7 +165,8 @@ async function getContainerBuildContextDirectory() {
               class="w-full p-2 mr-3 outline-none text-sm bg-charcoal-600 rounded-sm text-gray-700 placeholder-gray-700"
               required />
 
-            <Button on:click="{() => getContainerfileLocation()}">Browse...</Button>
+            <Button dataTestId="containerFilePathBrowse" on:click="{() => getContainerfileLocation()}"
```

Comment:
you could use the aria-label in the button component without adding a dataTestId.
Like
`<Button aria-label="containerFilePathBrowse"  ...`
then in the test
`screen.getByRole('button', { name: 'containerFilePathBrowse' `

Reply:
Nice, I didn't know this way of reference a component. Updated the code accordingly and rebased :)